src/services/arweave/SimpleArweaveDataProvider.ts

```diff
@@ -133,6 +141,7 @@ export class SimpleArweaveDataProvider implements ArweaveDataProvider {
     }
   }
   async validateArweaveId(id: string): Promise<ArweaveTransactionID> {
+    // todo: we should change this to a promise that checks for the existing of the tx id instead of just constructing a promise that rejects on class creation
```

Comment:
this could be as easy as callling `this.getTransactionHeaders(id)` and validating the status returns a `200`